web-access-app/src/main/java/edu/unc/lib/boxc/web/access/controllers/FullRecordController.java

```diff
@@ -220,6 +223,12 @@ public String getFullObjectView(String pidString) {
             recordProperties.put("dataFileUrl", dataFileUrl);
         }
 
+        String totalDownloadSize = null;
+        if (ResourceType.Work.nameEquals(resourceType)) {
```

Comment:
do you think it'd make sense to not set the property unless its a work, rather than returning null? Doesn't seem like it would impact the views as far as i can tell, but it doesn't matter too much.

Reply:
I think I find it more confusing when properties don't show up.